app/build.gradle.kts

```diff
@@ -1,5 +1,6 @@
 plugins {
     alias(libs.plugins.androidApplication)
+    id("com.google.android.gms.oss-licenses-plugin")
```

Comment:
Can we move this into the libs file?

Reply:
Umm, doesn't seem to work. So, the dependency is already in `gradle/libs.versions.toml` lines 19 & 61. However, when I try to remove this from here, the plugin then reports that the app doesn't have any open source licenses...

![no-license-message](https://github.com/user-attachments/assets/039f7f4d-d033-4ac0-83d0-cbba40ca12c8)

Reply:
Oh, IC - sorry.

Reply:
We can look into this in the future, it's not a big issue.